.github/workflows/build_test.yml

```
@@ -3,7 +3,7 @@ name: Build & Test
 on:
   workflow_dispatch:
   pull_request:
-    branches: ["main", "dev"]
+    branches: [ "main", "dev" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)